ts/screens/services/ServicesHomeScreen.tsx

```diff
@@ -136,7 +200,47 @@ class ServicesHomeScreen extends React.Component<Props, State> {
               </Text>
             </TabHeading>
           }
-        />
+        >
+          <ServicesNational
+            animated={{
+              onScroll: Animated.event(
+                [
+                  {
+                    nativeEvent: {
+                      contentOffset: {
+                        y: this.animatedScrollPositions[1]
+                      }
+                    }
+                  }
+                ],
+                { useNativeDriver: true }
+              ),
+              scrollEventThrottle: 8 // target is 120fps
+            }}
+            paddingForAnimation={true}
+            AnimatedCTAStyle={{
```

Comment:
Should **transform** object be defined one time and called within state currentTab?